crates/hir_ty/src/tests/macros.rs

```diff
@@ -773,6 +773,7 @@ mod clone {
 
 #[test]
 fn infer_derive_clone_in_core() {
+    // TODO: fix this
```

Comment:
The issue here is that there is not visibility for clone and it's by default visible when the trait is in scope. How can this be done?

Reply:
The latest commit "fixes" this by only including inherent candidates into the visibility check.